[MIG] shopfloor_reception_packaging_dimension/mobile: Migration to 16.0

rousseldenis opened the pull request:
Migrated shopfloor app from 18.0 and mobile part from 14.0

TDu committed:
Add shopfloor_reception_packaging_dimension_mobile

TDu committed:
Add shopfloor_reception_packaging_dimension

TDu committed:
sh_reception_packaging_dimension: refactor to allow module to extend

TDu committed:
sh_reception_packaging_dimension_mobile: refactor for extensibility

Commit:
[UPD] Update shopfloor_reception_packaging_dimension.pot

[BOT] post-merge updates

Commit:
[UPD] Update shopfloor_reception_packaging_dimension_mobile.pot

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Added translation using Weblate (Italian)

Translated using Weblate (Italian)

Currently translated at 100.0% (7 of 7 strings)

Translation: wms-14.0/wms-14.0-shopfloor_reception_packaging_dimension
Translate-URL: https://translation.odoo-community.org/projects/wms-14-0/wms-14-0-shopfloor_reception_packaging_dimension/it/

mymage committed:
Added translation using Weblate (Italian)

TDu committed:
sf_reception_packaging_dimension: fix test confirmation

OCA-git-bot committed:
[BOT] post-merge updates

grindtildeath committed:
[IMP] shopfloor_reception_packaging_dimension: pre-commit auto fixes

grindtildeath committed:
[MIG] shopfloor_reception_packaging_dimension: Migration to 18.0

simahawk committed:
shopfloor_reception_packaging_dimension: fix missing response validator

simahawk committed:
shopfloor_reception_packaging_dimension: fix init of class var

Such vars should be initialized when the component gets initialized.

simahawk committed:
shopfloor_reception_packaging_dimension: fix typo

simahawk committed:
shopfloor_reception_packaging_dimension: add hooks for packaging data

Commit:
[UPD] Update shopfloor_reception_packaging_dimension.pot

OCA-git-bot committed:
[BOT] post-merge updates

grindtildeath committed:
[IMP] shopfloor_reception_packaging_dimension: collect only needed dimensions

Commit:
[UPD] Update shopfloor_reception_packaging_dimension.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: stock-logistics-shopfloor-18.0/stock-logistics-shopfloor-18.0-shopfloor_reception_packaging_dimension
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-shopfloor-18-0/stock-logistics-shopfloor-18-0-shopfloor_reception_packaging_dimension/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (11 of 11 strings)

Translation: stock-logistics-shopfloor-18.0/stock-logistics-shopfloor-18.0-shopfloor_reception_packaging_dimension
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-shopfloor-18-0/stock-logistics-shopfloor-18-0-shopfloor_reception_packaging_dimension/it/

rousseldenis committed:
[IMP] shopfloor_reception_packaging_dimension: pre-commit autofixes

rousseldenis committed:
[MIG] shopfloor_reception_packaging_dimension: Migration to 16.0 (from 18.0)

rousseldenis commented:
/ocabot migration shopfloor_reception_packaging_dimension

rousseldenis committed:
[IMP] shopfloor_reception_packaging_dimension_mobile: pre-commit autofixes

rousseldenis committed:
[MIG] shopfloor_reception_packaging_dimension_mobile: Migration to 16.0

rousseldenis commented:
@phschmidt @lmignon @jbaudoux

mmequignon committed:
shopfloor_reception_packaging_dimension: Collect dimension optional when barcode empty

rousseldenis committed:
[FIX] shopfloor_reception_packaging_dimension_mobile: Use correct field for packaging weight

jbaudoux commented:
@nicolas-delbovier-acsone Can you keep only migration commits in this PR and also include imp/fix commits from https://github.com/OCA/stock-logistics-shopfloor/commits/18.0/shopfloor_reception_packaging_dimension

And then on top of that make another PR with your changes?

nicolas-delbovier-acsone committed:
[IMP] shopfloor_reception_packaging_dimension: format readme and index.html

nicolas-delbovier-acsone committed:
[FIX] shopfloor_reception_packaging_dimension: prevent session recovery noise in tests

Reset 'shopfloor_user_id' on the move line after the assertion to prevent
subsequent test steps from triggering a session recovery warning.

rousseldenis commented:
@nicolas-delbovier-acsone Could you approve ?

rousseldenis commented:
/ocabot merge nobump
